frontend/src/components/RecipeDraft.js

```diff
@@ -23,18 +59,38 @@ export default function RecipeDraft({recipe, onRecipeNameUpdated, onIngredientsU
                     onChange={onRecipeNameUpdated}
                 />
                 {ingredients.map(ingredient => (
-                    <IngredientEditor key={ingredient.id} ingredient={ingredient} onChange={onIngredientUpdated}/>
+                    <IngredientEditor
+                        key={ingredient.id}
+                        ingredient={ingredient}
+                        onChange={onIngredientUpdated}
+                        onDelete={onDeleteIngredient}
+                    />
                 ))}
-                <Button type="submit" >Speichern</Button>
+                <ButtonWrapper>
+                    <IconButton type="button" onClick={onAddIngredient}>
+                        <AddCircleOutlineOutlinedIcon/>
+                    </IconButton>
+                    <Button type="submit"><CheckOutlined/></Button>
```

Comment:
I had a hard time to understand why there are two buttons. The second button is necessary to submit the form, is it?

Reply:
The first button is to add a new Ingredient to a recipe, the second is to submit the whole Recipe and save it to the DB.